razorpay/CHANGELOG.md

```diff
@@ -5,6 +5,11 @@ Razorpay Prestashop Plugin Changelog
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [2.3.2] - 2019-12-09
+
+### Changed
+- Bug fixed for order captured
```

Comment:
Link to PR number or explain on the issue in more detail.

Reply:
> Fixed issue for reading the 'Payment action' configuration to create Razorpay order and set payment capture action correctly.

https://github.com/razorpay/razorpay-prestashop/pull/38